templates/create_movies_form.html

```diff
@@ -6,4 +6,45 @@
 <h1 class="mb-5">Create Movie Rating</h1>
 <p class="mb-3">Create a movie rating below</p>
 <!-- TODO: Feature 2 -->
+<form action="/movies" method="post">
+    <p class="mb-3">Title</p>
+    <input type="text" name="mov_title" id="mov_title" required>
+    <br>
```

Comment:
Is there a reason why it needs two breaks? But looking at the other codes so far it looks good.